docs/docs/integrations/vectorstores/databricks_vector_search.ipynb

```diff
@@ -0,0 +1,383 @@
+    "    index_name=index_name,\n",
+    "    primary_key=\"id\",\n",
+    "    embedding_dimension=emb_dim,\n",
+    "    embedding_vector_column=\"text_vector\",\n",
+    "    schema={\n",
+    "        \"id\": \"string\",\n",
+    "        \"text\": \"string\",\n",
+    "        \"text_vector\": \"array<float>\",\n",
+    "        \"source\": \"string\",\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "index.describe()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "047a14c9-2f06-4f74-883d-815b2c69786c",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from langchain.vectorstores import DatabricksVectorSearch\n",
+    "\n",
+    "dvs = DatabricksVectorSearch(\n",
+    "    index, text_column=\"text\", embedding=embeddings, columns=[\"source\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "951bd581-2ced-497f-9c70-4fda902fd3a1",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Add docs to the index"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "1e85f235-901f-4cf5-845f-5dbf4ce42078",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "dvs.add_documents(docs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "8bea6f0a-b305-455a-acba-99cc8c9350b5",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Similarity search"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "25c5a044-a61a-4929-9e65-a0f0462925df",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "query = \"What did the president say about Ketanji Brown Jackson\"\n",
+    "dvs.similarity_search(query)\n",
+    "print(docs[0].page_content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "46e3f41b-dac2-4bed-91cb-a3914c25d275",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Work with Delta Sync Index\n",
+    "\n",
+    "You can also use `DatabricksVectorSearch` to search in a Delta Sync Index. Delta Sync Index automatically syncs from a Delta table. You don't need to call `add_text`/`add_documents` manually."
```

Comment:
Maybe we add a link to the Databricks docs here for the definition of what a Delta Sync Index is and what is does on behalf of customers. Feel like we are glossing over what customers need to set up on their end if they want to use this index type

Reply:
Added a link.